doc/source/introduction.rst

```diff
@@ -0,0 +1,14 @@
+============
+Introduction
+============
+
+The **freud** library is a Python package for analyzing particle simulations.
+The package is designed to directly use numerical arrays of data, making it easy to use for a wide range of use-cases.
+The most common use-case of **freud** is for computing quantities on molecular dynamics simulation trajectories, but it can be used for analyzing essentially any type of simulation information.
```

Comment:
quantities _of_ or _from_?

Reply:
"can be used for analyzing essentially any type of simulation information" -- seems weirdly general, this doesn't really apply to CFD or weather simulations, for example. Perhaps "particle simulation"?